cpp/include/cudf/detail/aggregation/aggregation.cuh

```diff
@@ -216,12 +216,12 @@ struct identity_initializer {
  * @throw cudf::logic_error if column type is not fixed-width
  *
  * @param table The table of columns to initialize.
- * @param aggs A vector of aggregation operations corresponding to the table
+ * @param aggs A span of aggregation operations corresponding to the table
```

Comment:
Should we call out `host_span`?

Reply:
Honestly, I'd rather remove span completely:
`Aggregation operations corresponding to the table`
Not sure why the type is included in the docs, but I kept it as-is.